TeamCode/src/main/java/org/firstinspires/ftc/teamcode/ultimategoal/util/ModuleExecutor.java

```diff
@@ -37,6 +37,8 @@ public void run() {
                 lastUpdateTime = SystemClock.elapsedRealtime() - startTime;
             }
 
+            robot.fileDump.doTick();
```

Comment:
We should only file dump if WILL_FILE_DUMP, also I don't think we should tick both here and robot.

Reply:
whoops